doc/generate-api-docs.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
+
+cd "$(readlink -f "$(dirname "$0")/..")"
+
+rm -rf target/doc
+
+# Generate docs for core/alloc/std (x86_64-fortanix-unknown-sgx target)
+if [ -v RUST_DIR ]; then
+    PATCH="$(readlink -f "$(dirname "$0")/rust-std-edp-doc.patch")"
+    pushd $RUST_DIR
+    COMMIT=$(rustc --version -v|sed -n 's/^commit-hash: //p')
+    git checkout $COMMIT
+    git reset --hard $COMMIT
+    git submodule update --init --recursive
+    git apply < $PATCH
```

Comment:
A valid patch doesn't exist at this point

Reply:
Works for me? Is your nightly up to date?